spyder/plugins/ipythonconsole.py

```diff
@@ -418,7 +419,15 @@ def setup_page(self):
                           _("Height:")+"  ", " "+_("inches"),
                           'pylab/inline/height', min_=1, max_=20, step=1,
                           tip=_("Default is 4"))
-        
+        bbox_inches_box = newcb(
+                _("Use a tight layout for inline plots"),

```

Comment:
Hanging indent should be one indent level (four spaces) per PEP8, not two (eight), except in the case of a function definition. Spyder's automatic handling of it is in error, as I've pointed out in another issue a while ago.

Reply:
Thanks for the review. In the PEP8, it is said though that:
> The 4-space rule is optional for continuation lines.
>
> Optional:
>
> # Hanging indents *may* be indented to other than 4 spaces.
> foo = long_function_name(
>   var_one, var_two,
>   var_three, var_four)

source: https://www.python.org/dev/peps/pep-0008/#indentation 

Is this information somewhat outdated?

Reply:
Though, I also agree it is better to keep it to a consistent 4-space intent that eats less horizontal space than a 8-spaces indent, so I'm going to correct this anyway. I was just curious.

Reply:
Thanks for pointing that out. Of course, you would be much more aware than I if that was the case, so I presume that it is not. However, it seems prudent to adopt the standard four space convention for consistency with existing code and that from our many contributors—absent a strong reason to choose differently, which you might have.

Reply:
I'm not a professional programmer either... so I don't know better than you about this :)

I also agree with you. A 8 spaces hanging-intent doesn't make much sense if you think about it. In the first place, it is done for lack of horizontal space, while it eats 4 optional spaces..

So yeah, I think that Spyder automatic indentation, while not being wrong strictly speaking, is not opting for the smarter choice here.

Reply:
You're a heck of a lot closer to one than me, haha.

Particularly with the automatic indentation, , it seems only sensible that we go with the standard convention if we had to pick one, even if it isn't strictly binding. If I recall, the current 8 spaces was only supposed to happen for function definitions (a relatively small case), but either that was never implemented to differentiate between the two, or there is a bug in the implementation.